addons/fastroping/CfgVehicles.hpp

```diff
@@ -18,7 +18,7 @@ class CfgVehicles {
     };
     class ACE_Module: Module_F {};
     class ACE_moduleEquipFRIES: ACE_Module {
-        scope = 1;
+        scope = 2;
```

Comment:
I disabled that one because there is the Eden attribute for it. We decided to scrap modules in favor of Eden attributes some time ago.